farmdata2_modules/fd2_tabs/resources/CustomTableComponent.js

```diff
@@ -364,10 +364,10 @@ let CustomTableComponent = {
 
             for(let i = 0; i < this.rows.length; i++){
                 for(let j = 0; j < this.columns.length; j++){
-                    if(this.columns[j].visible){
+                    if(this.columns[j].visible && this.indexesToAction.includes(this.rows[i].id)){
                         if(typeof this.rows[i].data[j] === 'string'){
                             cleanHTML = this.rows[i].data[j].replaceAll(/(<p[^>]+?>|<p>|<\/p>|<br \/>)/img, "")
-                            cleanHTML = cleanHTML.replaceAll(/(\r\n|\n|\r)/gm, "-")
+                            cleanHTML = cleanHTML.replaceAll(/(\r\n|\n|\r)/gm, "")

```

Comment:
Here we should leave this as a `-` or maybe a " - " so that multiple lines do not get mashed together.  See the comment in the first row of the Seeding report table generated using the default dates for an example.  There each item should be separated with `-` or `" - "`

Reply:
This can be disregarded as it's primarily being addressed in PR #591. I just unfortunately made this feature branch from that one and not from main meaning those file changes can be found here as well.

Reply:
Sounds good. Though this now has created a conflict that will need to be resolved before this can be merged.  Just resolve that conflict accepting your prior changes and push the update.